### DISCUSSION TITLE
Add Docusaurus V3 to Framework Presets

### DISCUSSION THREAD

**Discussion by PaulRBerg** [vercel/community#5333] [Feature Question]
[Docusaurus V3](https://docusaurus.io/blog/releases/3.0) was announced on Oct 31, 2023.

It would be helpful to add it to the list Framework Presets list. Currently, only Docusaurus V1 and V2 are available.

**Comment by MorelSerge** [comment]
I've got a feeling this also breaks deployment of Docusaurus v3. Deployment works fine right now too, but every page that's not the root, throws a very brief 404 page (and also returns a 404 http code), which breaks crawlers etc ..

**Reply by MorelSerge** [reply]
I fixed this btw by setting the `cleanUrls` option to `true` in `vercel.json`
